### PR TITLE
doc: add base required files

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Comment by mhdawson** [comment]
Next will be README.md

File: LICENCE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+MIT License (MIT)
+=====================
+
+Copyright (c) 2017 Node.js package-maintenance  ollaborators
```
Comment (targos) [inline LICENCE.md]: Typo: collaborators

**Comment by mhdawson** [comment]
@targos thanks, fixed.

File: LICENCE.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+MIT License (MIT)
+=====================
+
+Copyright (c) 2017 Node.js package-maintenance collaborators
```
Comment (mcollina) [inline LICENCE.md]: I would just drop the copyright year.

Comment (mhdawson) [reply LICENCE.md]: I had meant to do that, Removing.

**Pull request merged**